### DISCUSSION TITLE
Face recognition using  EBGM     (Elastic     Bunch     Graph     Matching), Fisherfaces.

### DISCUSSION THREAD

**Discussion by akshitagupta15june** [akshitagupta15june/Face-X#421] [General]

**Comment by Hamza3226** [comment]
Hello, I am a GSSOC'21 participant and would like to contribute to this project.

**Comment by Preeti2095** [comment]
Can you assign this to me @akshitagupta15june?

**Comment by akshitagupta15june** [comment]
sure

**Comment by Preeti2095** [comment]
I would like to have collaboration on this issue. Can I ask for collaboration @akshitagupta15june?

**Comment by akshitagupta15june** [comment]
ok @Preeti2095

**Comment by akshitagupta15june** [comment]
let me know

**Comment by akshitagupta15june** [comment]
i am converting this issue to a discussion for you and your team

**Reply by Preeti2095** [reply]
OK.

**Comment by Preeti2095** [comment]
I want to implement this algorithm in collaboration. If anybody is interested let me know.

**Comment by shailendradix** [comment]
Hello, I am a SSOC'23 participant and would like to learn and contribute to this project.
